src/tasks.test.js

```diff
@@ -64,3 +64,44 @@ describe('add and remove', () => {
     expect(todoList.tasksArr).toHaveLength(2);
   });
 });
+
+describe('Edit test', () => {
+  test('Editing', () => {
+    const todoList = new TasksBluePrint();
+    const newTodo3 = {
+      id: '3',
+      description: 'task33',
+      completed: false,
+      index: 3,
+    };
+    todoList.add(newTodo3);
+    todoList.update(newTodo3.id, 'asd');
+    expect(todoList.tasksArr[2].description).toBe('asd');
+    expect(todoList.tasksArr).toHaveLength(3);
```

Comment:
- [ ] Kindly clarify why you are expecting `tasksArr` to have a length of 3, even though you are adding only one task in this test case?

Reply:
@gtekle because it follows up the workflow of the whole "describe()" functionalities. It's just an extra check to be sure everything is working as it is supposed to.